### PR TITLE
cleaned things up a bit with the cli

### PR DESCRIPTION
**Pull request by gamingdiamond982**
cleaned things up a bit and made it more like your typical cli

### CODE REVIEW

File: src/cli.py [hunk 1]
```diff
@@ -1,29 +1,45 @@
...
     # CLI start
     with LedgerServer('ledger.txt') as server:
         print(f"{_name} ver {_ver}")
+        print("run help for a list of commands")
+        print("or exit to leave the cli")
         acc = parse_account_id('@government')
         while True:
-            cmd = input(ps1(acc))+' '
-            if cmd.startswith('login '):
+            try:
```
Comment (JuhaJGamer) [inline src/cli.py:25]: this block is fine

File: src/cli.py [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
+
+            if cmd == '':
+                continue
+
+            if cmd.startswith('login'):
                 split = cmd.split()
                 acc_id = parse_account_id(split[1])
                 acc = acc_id
-            elif cmd.startswith('exit '):
+            elif cmd.startswith('exit'):
```
Comment (JuhaJGamer) [inline src/cli.py:41]: I dont like this tbf.

No cli says goodbye

**Review by JuhaJGamer — CHANGES_REQUESTED** [review]
Take off the goodbye smh

File: src/cli.py (hunk 2 above)
Comment (gamingdiamond982) [reply src/cli.py:41]: I've seen plenty of CLI's say goodbye but ok

Comment (JuhaJGamer) [reply src/cli.py:41]: shit clis smh

**Review by JuhaJGamer — APPROVED** [review]

**Pull request merged**